DMTN-222.tex

```diff
@@ -454,7 +454,302 @@ \section{Batch Process Submission (BPS) Templates}
 \section{Full Calibration Example}
 \label{sec:example}
 
-TODO: DM-48520
+This section contains the commands used to generate, certify, chain, and verify calibrations for LSSTComCam DP1 processing.
+This is adapted from the script attached to DM-48520.
+
+Note that this is not a complete construction of all calibrations; for this we used pre-existing linearity, ptc, and bfk.
+
+In addition to the calibration building, this includes some checkpoints to download mosaic images for visual spot-checking to ensure that nothing has gone terribly wrong before continuing on all the way to verification reports.
+
+\begin{verbatim}
+
+export USER_CALIB_PREFIX=""
+export INSTRUMENT=LSSTComCam
+export TICKET=DM-48520
+export REPO=/repo/main
+export RAW_COLLECTION=LSSTComCam/raw/all
+export CALIB_COLLECTIONS=LSSTComCam/calib/DM-48650,LSSTComCam/calib/DM-47447/gainFixup/ptc.20241107a,LSSTComCam/calib/DM-46360/isrTaskLSST/linearizer.20240926a,LSSTComCam/calib/DM-46360/isrTaskLSST/bfk.20240926a,LSSTComCam/calib/DM-46360/isrTaskLSST/ptc.20240926a
+export TAG=DP1
+export RERUN=20250207a
+export BOOTSTRAP_RUN_NUMBER=1
+
+# These are certification env vars
+export START_DATE=1970-01-01T00:00:00
+export CERT_RERUN=20250207a
+
+export SELECTION_BIAS="instrument='LSSTComCam' and ((exposure.day_obs=20241208 and exposure.seq_num >= 696) or (exposure.day_obs=20241206 and exposure.seq_num >=427)) and exposure.observation_type='bias'"
```

Comment:
Do these need to have the instrument hard coded, or can they use the `instrument='$INSTRUMENT'` form used below?